src/common/utils/document.ts

```diff
@@ -0,0 +1,5 @@
+import { WrappedDocument } from "@govtechsg/open-attestation";
```

Comment:
eslint doesnt complain on this ? 2 imports of the same lib in 2 lines (not that I care that much)

Reply:
eslint is stupid, thats why we have you! (=